components/BottomNavigation/BUILD

```diff
@@ -95,6 +95,15 @@ mdc_objc_library(
     ],
 )
 
+mdc_extension_objc_library(
+    name = "Theming",
+    deps = [
+        ":BottomNavigation",
+        "//components/ShadowElevations",
+        "//components/schemes/Container",
```

Comment:
The CocoaPods podspec indicates that the dependencies should include the Color and Typography schemes. Please make sure they match. I suspect CocoaPods is correct (explicit dependencies).

Reply:
Done